test/services/return-requirements/submit-setup.service.test.js

```diff
@@ -94,4 +96,31 @@ describe('Submit Setup service', () => {
       })
     })
   })
+  describe('with different setups', () => {
```

Comment:
Just a NIT. We like to leave a line between the `describe()` blocks.

```suggestion

  describe('with different setups', () => {
```